builtin/providers/aws/resource_aws_ecs_service.go

```diff
@@ -156,6 +156,8 @@ func resourceAwsEcsServiceRead(d *schema.ResourceData, meta interface{}) error {
 	}
 
 	if len(out.Services) < 1 {
+		log.Printf("[DEBUG] Removing ECS service %q because it's gone", service.ServiceArn)
```

Comment:
> I wonder why Travis didn't pick this up btw.

I was wondering the same thing. Very strange :confused: Oh well, it's fixed, thanks for the follow up PR :smile: